src/wp-includes/functions.php

```diff
@@ -5261,7 +5261,7 @@ function wp_widgets_add_menu() {
 	}
 
 	$menu_name = __( 'Widgets' );
-	if ( wp_is_block_theme() ) {
+	if ( wp_is_block_theme() || current_theme_supports( 'block-template-parts' ) ) {
```

Comment:
We need this to avoid conflicts between the `Customize` and `Widgets` menu positions.